services/registration/controller/controller.go

```diff
@@ -124,41 +124,41 @@ func CreateCurrentUserRegistration(w http.ResponseWriter, r *http.Request) {
 	err = service.CreateUserRegistration(id, user_registration)
 
 	if err != nil {
-		panic(errors.UnprocessableError(err.Error()))
+		panic(errors.InternalError(err.Error(), "Could not create user registration.\n"+err.Error()))
 	}
 
 	err = service.AddApplicantRole(id)
 
 	if err != nil {
-		panic(errors.UnprocessableError(err.Error()))
+		panic(errors.InternalError(err.Error(), "Could not add applicant role.\n"+err.Error()))
```

Comment:
Don't need a newline here and don't need to append `err.Error()` since it will be included in the `RawError` field.